src/main/java/org/lanternpowered/server/game/registry/type/item/ItemRegistryModule.java

```diff
@@ -646,6 +649,10 @@ public void registerDefaults() {
         register(323, builder()
                 .translation("item.sign.name")
                 .maxStackQuantity(16)
+                .behaviors(pipeline -> pipeline
+                        .add(WallOrStandingPlacementBehavior.ofTypes(
+                                () -> BlockTypes.WALL_SIGN,
+                                () -> BlockTypes.STANDING_SIGN)))
```

Comment:
You need some sort of behavior to send a `MessagePlayOutOpenSign` here.

Reply:
Yeah, right now it's right click to open a sign.

Reply:
I noticed that, it's a nice touch.